cortex-mixin/alerts/alerts.libsonnet

```
@@ -352,6 +352,30 @@
         },
       ],
     },
+    {
+      name: 'cortex_distributor_inflight_push_request_alert',
+      rules: [
+        {
+          alert: 'CortexDistributorReachingInflightPushRequestLimits',
+          expr: |||
+            (
+                (cortex_distributor_inflight_push_requests / ignoring(limit) cortex_distributor_instance_limits{limit="max_inflight_push_requests"})
+                and ignoring (limit)
+                (cortex_distributor_instance_limits{limit="max_inflight_push_requests"} > 0)
+            ) > 0.9
```

I would set it to 80%.
```suggestion
            ) > 0.8
```